src/Controls/src/Core/Platform/Android/Extensions/TextViewExtensions.cs

```diff
@@ -67,12 +68,11 @@ internal static void SetLineBreakMode(this TextView textView, LineBreakMode line
 					break;
 				case LineBreakMode.TailTruncation:
 
-					// Leaving this in for now to preserve existing behavior
```

Comment:
The existing behavior is different from Xamarin.Forms.
![issue-13981-forms](https://user-images.githubusercontent.com/6755973/236147825-784a5a35-913a-4c4e-b01f-4dd3cb392fdf.PNG)

In this PR we apply changes to only force the single line with the Button because has no MaxLines property.